entries/s/subscribestar.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Patreon": {
+    "domain": "subscribestar.com",
+    "url": "https://www.subscribestar.com/",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```